packages/react/src/components/Modal/DeleteConfirmationModal.tsx

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import { Modal, ModalSize } from ":/components/Modal/index";
+import { useCunningham } from ":/components/Provider";
+import { Button } from ":/components/Button";
+import { DecisionModalProps } from ":/components/Modal/ModalProvider";
+
+export type DeleteConfirmationModalProps = DecisionModalProps;
+
+export const DeleteConfirmationModal = ({
+  title,
+  children,
+  onDecide,
+  ...props
+}: DeleteConfirmationModalProps) => {
+  const { t } = useCunningham();
+  return (
+    <Modal
+      title={title ?? t("components.modals.helpers.delete_confirmation.title")}
+      titleIcon={<span className="material-icons clr-danger-600">delete</span>}
+      size={ModalSize.SMALL}
+      actions={
+        <>
+          <Button
+            color="secondary"
+            fullWidth={true}
+            onClick={() => onDecide(null)}
+          >
```

Comment:
With the meeting with Axel, I wonder if we should not have a "danger" variant of the Button then use it here.